pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -906,6 +906,15 @@ class Build {
                 }
             }
 
+            try {
+                // Clean any old workspace build folder as it contains 90k+ files which could slow git clean -fdx
+                // https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1553
+                context.echo("Cleaning any old workspace build from ${env.WORKSPACE}/workspace/build/src/build, as git clean -fdx can take a very long time to do the same.")
```

Comment:
Can we add a link to the issue where this was reported? Also is this a better place to put this logic compared to prepareWorkspace in build.sh?

Reply:
agree

Reply:
This scm checkout of openjdk-build occurs right up front of the job, prior to the prepareWorkspace.sh being called... (form openjdk-build)